.ci/ci-go-static-checks.sh

```diff
@@ -69,12 +69,22 @@ install_package github.com/golang/lint/golint
 
 echo Doing go static checks on packages: $go_packages
 
+echo "Running misspell..."
 go list -f '{{.Dir}}/*.go' $go_packages |\
     xargs -I % bash -c "misspell -error %"
+
+echo "Running go vet..."
 go vet $go_packages
+
+echo "Running gofmt..."
 go list -f '{{.Dir}}' $go_packages |\
-    xargs gofmt -s -l | wc -l |\
-    xargs -I % bash -c "test % -eq 0"
+    xargs gofmt -s -l | tee /dev/tty | \
```

Comment:
why would it be better to use the tty command? :) [controlling terminal](http://tldp.org/HOWTO/Text-Terminal-HOWTO-7.html#ss7.3) of the process group (/dev/tty) Vs the path of the tty if stdin is indeed a terminal (tty)

It's also not very handy to do a onliner with tty. tty works by doing a readlink() on /proc/self/fd/0, which is of course a pipe when following |, so we can't use it on that line:

```
$ ls /etc/passwd |  echo `tty` 
not a tty
```

and so:

```
$ ls /etc/passwd | tee `tty` > /dev/null
$ ls /etc/passwd | tee /dev/tty > /dev/null
/etc/passwd
$
```